scipy-1.0/ckdtree.tex

```diff
@@ -15,8 +15,8 @@
 
 In 2015, SciPy added the \texttt{sparse\_distance\_matrix} routine for generating 
 approximate sparse distance matrices between \texttt{KDTree} objects by ignoring 
-all distances that exceed a user-provided value. Also, this routine is not 
-limited to the conventional L2 (Euclidean) norm but supports any Minkowski 
+all distances that exceed a user-provided value. This routine is not 
+limited to the conventional L2 (Euclidean) norm, but supports any Minkowski 
```

Comment:
Removing "Also" is fine, but this new comma is not appropriate because "supports any Minkowski $p$-norm between 1 and infinity" is not an independent clause. 
https://www.grammarly.com/blog/comma-before-but/

Reply:
Grammarly doesn't provide an example comparable to this where the entire verb phrase is coordinated by but. I would go for whichever reads better over a textbook prescription here. Which do you think reads better?

Reply:
Someone else commented on the grammarly blog about the absence of this verb phrase coordination case. I've adopted your/their preference in any case.